beacon_node/network/src/sync/manager.rs

```diff
@@ -1186,10 +1189,9 @@ impl<T: BeaconChainTypes> SyncManager<T> {
         block: RpcEvent<Arc<SignedBeaconBlock<T::EthSpec>>>,
     ) {
         if let Some(resp) = self.network.on_blocks_by_range_response(id, peer_id, block) {
-            self.on_range_components_response(
+            self.on_block_components_by_range_response(
```

Comment:
I often get confused here as we have layers of `on_xxx_by_range_response` functions, what about renaming:
- `resp` to `result`
- `self.on_block_components_by_range_response` to `self.on_range_components_rpc_result`, and add a comment describing it:

```suggestion
            // Process it further if the response results in a completed request or a failure that needs to be handled.
            self.on_range_components_rpc_result(
```

Reply:
also might help to add some comment to `self.network.on_blocks_by_range_response`. I wrote these notes while reviewing:

```rust
    /// Processes the `BlocksByRange` response (`RpcEvent`), depending on the event, it may:
    /// - update the active request state; or do nothing.
    /// - complete the active request and return an `Ok` response containing a list of response objects.
    /// - propagate the error if necessary.
    pub(crate) fn on_blocks_by_range_response(
        &mut self,
        id: BlocksByRangeRequestId,
        peer_id: PeerId,
        rpc_event: RpcEvent<Arc<SignedBeaconBlock<T::EthSpec>>>,
    ) -> Option<RpcResponseResult<Vec<Arc<SignedBeaconBlock<T::EthSpec>>>>> {
```

Reply:
Added a note in the PR body